modules/compute/vm-instance/main.tf

```diff
@@ -94,7 +94,8 @@ data "google_compute_image" "compute_image" {
 
 resource "null_resource" "image" {
   triggers = {
-    image   = var.instance_image.family,
+    name    = try(var.instance_image.name, null),
+    family  = try(var.instance_image.family, null),
     project = var.instance_image.project
```

Comment:
Let's wrap project lookup into `try` as well, for a good measure.
Please consider adding a validation test, I'm OK if you decide to do it as a separate PR.

Reply:
I am not, please do add a validation test as part of this PR

Reply:
Done